cli/command/image/tree.go

```diff
@@ -0,0 +1,244 @@
+package image
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/docker/cli/cli/command"
+
+	"github.com/containerd/platforms"
+	"github.com/docker/docker/api/types/filters"
+	imagetypes "github.com/docker/docker/api/types/image"
+	"github.com/docker/docker/pkg/stringid"
+	"github.com/docker/go-units"
+	"github.com/fatih/color"
+)
+
+type treeOptions struct {
+	all     bool
+	filters filters.Args
+}
+
+func runTree(ctx context.Context, dockerCLI command.Cli, opts treeOptions) error {
+	images, err := dockerCLI.Client().ImageList(ctx, imagetypes.ListOptions{
+		All:            opts.all,
+		ContainerCount: true,
```

Comment:
Needs: https://github.com/moby/moby/pull/47501

Reply:
We have, but it's only filled if the `ContainerCount` option is set (we can still change that though).

Reply:
Should I remove the `Usage` column for now?